scripts/dev_cli.sh

```diff
@@ -298,7 +298,7 @@ cmd_build() {
         --env RUSTFLAGS="$rustflags" \
         --env TARGET_CC="$target_cc" \
         "$CTR_IMAGE" \
-        cargo build --all "$features_build" \
+        cargo build --all $features_build \
```

Comment:
I changed the features_build var type to an array. It should pass the checks now.

Reply:
This could be solved by using bash parameter substitution to pass null string (and not quotes) to cargo if  features_build is null or unset. For example:

```
-        cargo build --all "$features_build" \
+        cargo build --all ${features_build:+"$features_build"} \
```
Changing features_build to an array should also work.